package.json

```diff
@@ -136,12 +138,13 @@
     "testcafe-browser-tools": "2.0.23",
     "testcafe-hammerhead": "24.5.18",
     "testcafe-legacy-api": "5.1.4",
-    "testcafe-reporter-dashboard": "0.2.6-rc.2",
+    "testcafe-reporter-dashboard": "0.2.6-rc.3",
```

Comment:
Just a note, we should not release `testcafe` with `testcafe-reporter-dashboard` RC version